xbmc/games/controllers/types/PortNode.cpp

```diff
@@ -70,6 +70,20 @@ CControllerNode& CPortNode::GetActiveController()
   return invalid;
 }
 
+bool CPortNode::SetActiveController(const std::string& controllerId)
+{
+  for (unsigned int i = 0; i < m_controllers.size(); ++i)
```

Comment:
```suggestion
  for (size_t i = 0; i < m_controllers.size(); ++i)
```
Not sure if there would be a comparison warning here or not.

Reply:
Our multitude of platforms is always triggering comparison warnings. Might as well avoid that.